pulser-core/pulser/channels.py

```diff
@@ -450,10 +453,16 @@ def __repr__(self) -> str:
             )
             if cast(int, self.max_targets) > 1:
                 config += f", Max targets: {self.max_targets}"
-        config += f", Basis: '{self.basis}'"
+        config += (
+            f", Clock period: {self.clock_period} ns"
+            f", Minimum pulse duration: {self.min_duration} ns"
+        )
+        if self.max_duration is not None:
+            config += f", Maximum pulse duration: {self.max_duration} ns"
```

Comment:
It might be more consistent to decide whether to display None parameters or not in general (for example, `max_abs_detuning` is included even if None, as opposed to `max_duration`).

Reply:
I considered this and I agree with it being more consistent. My reasoning beyond leaving it like this was:
- `max_abs_detuning` and `max_amp` are central parameters that greatly influence the sequence creating and thus should always be represented
- `max_duration` is virtually never relevant so I figured it would just take unnecessary space when not defined
- `mod_bandwidth` doesn't actually invalidate the sequence creation, so when it is not defined it doesn't need to be represented either

I understand that these justifications are weak, I think in the end I was just trying not to bloat the repr unnecessarily.